File: Pattern12.go
package main

import "fmt"

func main() {
	var n int
	fmt.Println("Enter the row number:")
	fmt.Scanf("%d", &n)

	for i := 1; i <= n; i++ {
		for j := i; j <= n; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}

File: Pattern12.c
// *****
// ****
// ***
// **
// *

#include <stdio.h>

void main()
{
    int n;
    printf("Enter the number of row:");
    scanf("%d", &n);
    for (int i = 1; i <= n; i++)
    {
        for (int j = i; j <= n; j++)
        {
            printf("*");
        }
        printf("\n");
    }
}